src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -744,7 +744,7 @@ void checkMemberStateOnConstructorExit(MethodSymbol constructor, Symbol member,
                     return;
                 }
 
-                var memberState = state[slot];
+                var memberState = GetState(ref state, slot);
```

Comment:
Just curious, did you consider an API like `state.Get(this, slot)` or even `state[this, slot]`?

Reply:
Yes, but one of the main things is that I didn't want to pass `this` at all the call sites